src/modules/user/user.usecase.spec.ts

```diff
@@ -217,6 +248,244 @@ describe('User use cases', () => {
       });
     });
   });
+
+  describe('changeUserEmailById', () => {
+    it('should change the user email successfully', async () => {
```

Comment:
Using "Should" is pretty valid, however, let's follow the same naming convention that is being used across the other tests. We should follow the same pattern even if it sounds too repetitive.

**"When X, then X"**   Check the tests made for Resseting a user.